packages/connector/src/MosDevice.ts

```diff
@@ -1539,6 +1539,13 @@ export class MosDevice implements IMOSDevice {
 			throw new Error('Buddy server cannot respond because main server is available')
 		}
 
+		if (
+			reply.mos.roAck &&
+			reply.mos.roAck.roStatus === 'Main server back on line but not yet updated with most recent data'
```

Comment:
That is a quite exact phrase, should we perhaps change this into a more lenient regex, to account for possible future changes in the reply from ENPS?

```suggestion
			// ENPS replies with "Main server back on line but not yet updated with most recent data"
			`${reply.mos.roAck.roStatus }`.match(/Main server.*on line /i)''
```

Reply:
I think it's a false hope to be more lenient. If ENPS chooses to change it in a slightly different way it will break anyway. And this message is checked on _any_ roAck we receive so possible performance hit too.